badge/badge.css

```diff
@@ -0,0 +1,69 @@
+@import url(/style.css);
+@import url(/avatar\avatar.css);
+
+.icon-container{
+    width: 30%;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    align-items: center;
+}
+.badge-icon{
+    width: 2rem;
+    height: 2rem;
+}
+.badge-on-icon{
+    position: relative;
+}
+.badge{
+    position: absolute;
+    color: white;
+    top: -8px;
+    border-radius: 50%;
+    width: 0.7rem;
+    height: 1.2rem;
+    padding: 0.05rem;
+    text-align: center;
+}
+.badge-red{
+    background-color: red;
+}
+.badge-green{
+    background-color: green;
+}
+.badge-black{
+    background-color: black;
+}
+.text{
+    background-color: cadetblue;
+    width: max-content;
+    height: max-content; 
+    padding: 0.3rem;
+    border-radius: 8px;
+    margin: 0;
+}
```

Comment:
Add these colors in a variable and use it in whole project. It will help you customise things in long run